src/coreclr/jit/importer.cpp

```diff
@@ -15789,6 +15949,9 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         op2->gtType = TYP_I_IMPL;
                     }
                     else
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks